src/main/java/com/checkout/disputes/four/DisputesClientImpl.java

```diff
@@ -1,21 +1,14 @@
 package com.checkout.disputes.four;
 
-import com.checkout.AbstractClient;
 import com.checkout.ApiClient;
 import com.checkout.CheckoutConfiguration;
 import com.checkout.SdkAuthorizationType;
-import com.checkout.common.FileDetailsResponse;
-import com.checkout.common.FileRequest;
-import com.checkout.common.IdResponse;
 
 import java.util.concurrent.CompletableFuture;
 
-public class DisputesClientImpl extends AbstractClient implements DisputesClient {
+public class DisputesClientImpl extends com.checkout.disputes.DisputesClientImpl implements DisputesClient {
```

Comment:
Don't really like this approach, but I think it's a great improvement and a step forward in terms of code reuse on this module. 🤙